examples/wrap/style.css

```diff
@@ -33,16 +32,22 @@
   margin: 0.5rem;
 }
 
+#n {
```

Comment:
same as everywhere else, `n` should be named something less generic.

Reply:
Done.